recipes-bsp/u-boot/files/0014-iot2050-add-the-m2-variant-to-the-of_list.patch

```diff
@@ -0,0 +1,30 @@
+From 0c6386ebe10b20269492edf1a482f687e271d9bd Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Tue, 8 Mar 2022 13:47:32 +0800
+Subject: [PATCH 14/14] iot2050: add the m2 variant to the of_list
+
+has already use the of_list to create the node,
+in order to support m2 variant, add the m2 variant
+dtb to the list
```

Comment:
I think patch 10, 11 and 14 could be combined to provide the feature addition in one step and avoiding that things are broken between them.